.vscode/settings.json

```diff
@@ -84,5 +84,6 @@
     },
     "jest.autoRun": {
       "watch": false
-    }
+    },
+    "sarif-viewer.connectToGithubCodeScanning": "off"
```

Comment:
This is just annoying - VS Code keeps asking me if I want to connect to gitHub for sarif results. Every single  time I open VS Code. The answer is no! We're not even using sarif  output in Meadowlark right now.